### ISSUE TITLE
Settle the bibliography

### ISSUE THREAD

**Issue by LinguList** [glottobank/lexibank-data#63]
Currently, most of our bibliographic entries are "misc", based on the fact that we get them in spreadsheets and not in bibtex. Ideally, we would be able to link from available bibtex datasets, using keys. I did that for data I contributed, using the [EvoBib](http://bibliography.lingpy.org) database, for which we have a script to download the bibtex, using an AuthorYear key. 

We could organize a list of pending bibtex entries for the misc-fields, which would be manually updated. But for this, we need a command that creates a file of all bad bibliographies. And when creating the cldf, the command should lookup, whether it finds a global bibliography that is suitable, maybe in a general bibliography file that we use. 

Any more concrete ideas are much appreciated, as I don't really know how to handle this best. But I'd say that having good bibliographies in LexiBank is a definite desideratum, even if it is tedious and no fun to organize them...

**Comment by xrotwang** [comment]
I guess EvoBib should be included in Glottolog, and Glottolog reference IDs should be used to link to sources.

**Comment by LinguList** [comment]
There's probably already a significant overlap. Bai data, e.g., is [Allen2007](http://bibliography.lingpy.org/raw.php?key=Allen2007) vs. [Bryan Allen 2007](http://glottolog.org/resource/reference/id/29664).

The question is how we handle those cases where we need to hardcode bibliographic entries, because they are missing.

And how we can make this process of assigning sources more straightforward, e.g., by having a master-list, where missing links to Glottolog are shown, etc., so that one can at some point just go through the bibliography and correct the things that are missing in one run.

**Comment by LinguList** [comment]
Super, that means, apart from that global bibtex file, and eventually some test on which keys are still missing (which could be done via the API, or actually is already done when writing the readme), we only need the references/references.bib folder globally listing all entries, right?

**Comment by xrotwang** [comment]
Yes. I'm not sure whether this bibtex file should live in the `clld/glottolog` repository right away - maybe wait some time until it stabilizes - but eventually, that's where it should move.
